commands/generate.js

```diff
@@ -94,7 +95,20 @@ function generateApp(program) {
     .catch(function (err) {
       return deferred.reject(err);
     });
-  });
+  };  
+  
+  
+  if (platforms.length === 1 && platforms[0] === 'edgeextension')

```

Comment:
I guess I could move this up above and early out instead of wrapping this in another function call